content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -191,68 +211,78 @@ EOF
 
 Some points to note:
 
-- usage has to be 'client auth'
-- request is the base64 encoded value of the CSR file content. You can use this command to get that ```cat john.csr | base64 | tr -d "\n"```
+- `usages` has to be '`client auth`'
+- `request` is the base64 encoded value of the CSR file content.
+  You can get the content using this command: ```cat john.csr | base64 | tr -d "\n"```
 
-### Approve Certificate Request
+### Approve certificate signing request
 
 Use kubectl to create a CSR and approve it.
 
-Get the list of CSRs
-```
+Get the list of CSRs:
+
+```shell
 kubectl get csr
 ```
 
-Approve the CSR
-```
+Approve the CSR:
+
+```shell
 kubectl certificate approve john
 ```
 
-### Get the Certificate
+### Get the certificate
 
-Retrieve the Certificate from the CSR.
+Retrieve the certificate from the CSR:
 
-```
+```shell
 kubectl get csr/john -o yaml
 ```
 
-The Certificate value is in Base64-encoded format under status.certificate.
+The certificate value is in Base64-encoded format under `status.certificate`.
 
-### Create Role and Role Binding
+### Create Role and RoleBinding
 
-You get the Certificate already. Now it is time to define the Role and Role Binding for this user to access Kubernetes Cluster resources.
+You get the certificate already. Now it is time to define the Role and RoleBinding for
+this user to access Kubernetes cluster resources.
 
-This is a sample script to create role for this new user
-```
+This is a sample script to create a Role for this new user:
+
+```shell
 kubectl create role developer --verb=create --verb=get --verb=list --verb=update --verb=delete --resource=pods
 ```
 
-This is a sample script to create role binding for this new user
-```
+This is a sample command to create a RoleBinding for this new user:
+
+```shell
 kubectl create rolebinding developer-binding-john --role=developer --user=john
 ```
 
-### Add to KubeConfig
+### Add to kubeconfig
+
+The last step is to add this user into the kubeconfig file.
+We assume the key and certificate files are located at "/home/vagrant/work/".
```

Comment:
nit: replace "We"
This example assumes the key and certificate files are located at ...